ajax/libs/isso/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "isso",
+  "filename": "embed.min.js",
+  "version": "0.1.0",
+  "description": "A Disqus Alternative",
+  "homepage": "https://posativ.org/isso/",
+  "keywords": [
+    "isso",
+    "Disqus",
+    "comments"
+  ],
+  "author": {
+    "name": "posativ",
+    "url": "https://github.com/posativ/"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/posativ/isso.git"
+  }
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/SnorlaxYum/isso-js.git",
```

Comment:
U couldn't get the js files from the official repo directly. That's why I created another repo